redshift_admin.view.lkml

```diff
@@ -335,3 +355,91 @@ view: data_loads {
       ;;
   }
 }
+
+view: recent_plan_steps {
+  #Recent is last 1000 queries
+  derived_table: {
+    # Insert into PDT because redshift won't allow joining certain system tables/views onto others (presumably because they are located only on the leader node)
+    persist_for: "1 hour"
+    sql:
+        SELECT
+        query, nodeid,
+        substring(regexp_substr(plannode, 'XN ([A-Z][a-z]+ ?)+'),3) as oper,
```

Comment:
I see two metacharacters next to one another: `+ ?`. Is Redshift picking up the space between them as an implicit `\\s`? Perhaps I'm missing something.